great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -780,6 +776,7 @@ def get_domain_records(  # noqa: C901 - 24
 
         return selectable
 
+    @public_api
     def get_compute_domain(
         self,
```

Comment:
Same comment - double check the rendering. Might need colons after each arg. I could be wrong though.